riff-raff/app/views/deploy/viewDeploy.scala.html

```diff
@@ -25,7 +26,7 @@
             <h2>Deploy of @record.buildName build @record.buildId@record.metaData.get("branch").map{ branch=> [@branch]} in @record.stage.name</h2>
 
             @record.metaData.get(deployment.Record.RIFFRAFF_HOSTNAME).map { hostname =>
-                <small>This deploy @if(record.isRunning) {is running} else {ran} on @hostname</small>
+                <small>This deploy @if(record.isRunning) {is running} else {ran} on @hostname at @record.time.toString(DateFormats.Short)</small>
```

Comment:
Does this look weird when running (i.e. "This deploy is running on blah at 23/5/18 14:37")?

Reply:
Yeah, I think this looks weird.

Reply:
How about simplifying to "This deploy started at <date/time> on <host>"